lib/autoupdate/agent/updater.go

```diff
@@ -350,6 +357,8 @@ type OverrideConfig struct {
 	AllowOverwrite bool
 	// AllowProxyConflict when proxies in teleport.yaml and update.yaml are mismatched.
 	AllowProxyConflict bool
+	// SELinuxFlagSet specifies whether the user explicitly toggled SELinux behavior.
+	SELinuxFlagSet bool
```

Comment:
```suggestion
	SELinuxChanged bool
```
Flags means something different in the context of this struct